incubator/hnc/internal/validators/object_test.go

```diff
@@ -142,7 +143,14 @@ func TestInheritedFromLabel(t *testing.T) {
 }
 
 func TestUserChanges(t *testing.T) {
-	f := forest.NewForest()
+	// Construct a forest with a <- b ('a' is the parent). Create a secret object
+	// called 'conflict' in namespace 'b'.
+	f := foresttest.Create("-a")
+	inst := &unstructured.Unstructured{}
+	inst.SetName("conflict")
+	inst.SetNamespace("b")
+	inst.SetGroupVersionKind(schema.GroupVersionKind{Group: "", Version: "v1", Kind: "Secret"})
+	f.Get("b").SetOriginalObject(inst)
```

Comment:
Since this has a different setup for any other case in `TestUserChanges`, maybe make a new top-level function? That way, you can also test a few different topologies (e.g. have a conflict in a child and a grandchild).